src/Uno.UI.Runtime.Skia.Win32/Builder/HostBuilderExtensions.cs

```diff
@@ -7,4 +7,10 @@ public static IUnoPlatformHostBuilder UseWin32(this IUnoPlatformHostBuilder buil
 		builder.AddHostBuilder(() => new Win32HostBuilder());
 		return builder;
 	}
+
+	public static IUnoPlatformHostBuilder UseWin32(this IUnoPlatformHostBuilder builder, bool preloadMediaPlayer)
+	{
+		builder.AddHostBuilder(() => new Win32HostBuilder().PreloadMediaPlayer(preloadMediaPlayer));
```

Comment:
Let's create a sub-builder for that one, we'll otherwise end up having too many parameters in the long run.

Something like

```
.UseWin32(b => b
   .MediaPlayer(mp => mp
      mp.EnablePreload()
   )
```